packages/block-editor/src/components/block-list/content.scss

```diff
@@ -260,7 +260,7 @@ _::-webkit-full-page-media, _:future, :root .has-multi-selection .block-editor-b
 	}
 
 	// Reusable blocks parent border.
-	&.is-reusable.has-child-selected::after {
+	&.is-reusable:has(.is-selected)::after {
```

Comment:
Reusable blocks are no longer editable in this way inline, so this rule could maybe be removed.